pkg/scheduler/scheduler.go

```diff
@@ -599,14 +600,15 @@ func (sched *Scheduler) scheduleOne() {
 			ObjectMeta: metav1.ObjectMeta{Namespace: assumedPod.Namespace, Name: assumedPod.Name, UID: assumedPod.UID},
 			Target: v1.ObjectReference{
 				Kind: "Node",
-				Name: suggestedHost,
+				Name: scheduleResult.SuggestedHost,
 			},
 		})
 		metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
 		if err != nil {
 			klog.Errorf("error binding pod: %v", err)
 			metrics.PodScheduleErrors.Inc()
 		} else {
+			klog.V(1).Infof("success binded pod: %v/%v on node %v, evaluated %d nodes and %d feasible nodes", assumedPod.Namespace, assumedPod.Name, scheduleResult.SuggestedHost, scheduleResult.EvaluatedNodesNumber, scheduleResult.FeasibleNodesNumber)
```

Comment:
`V(1)` will generate lots of log records. Would suggest to increase the verbose level.

And suggest: s/success binded pod/pod gets bound successfully/

Reply:
Done, thanks HuangWei. :)